doc/cephfs/fs-volumes.rst

```diff
@@ -364,6 +364,40 @@ This command removes the subvolume and its contents. This is done in two steps.
 First, the subvolume is moved to a trash folder. Second, the contents of that
 trash folder are purged asynchronously.
 
+The progress made by this asynchronous purge can be checked by viewing the
+output of ``ceph status`` command. When asynchronous purge is ongoing, a
+progress bar is printed at the bottom of the ``ceph status`` output (probably
+along with other progress bars) in a section called "progress". Following is
```

Comment:
I would remove `(probably along with other progress bars)`

Reply:
This to tell the user that that section will probably contain other progress bars as well.